File: lib/models/employee.dart
class Employee {
  final String id;
  final String fullName;
  final String dob;
  final String fatherName;
  final String fatherOccupation;
  final String aadhar;
  final String address;

  Employee({
    required this.id,
    required this.fullName,
    required this.dob,
    required this.fatherName,
    required this.fatherOccupation,
    required this.aadhar,
    required this.address,
  });

  factory Employee.fromJson(Map<String, dynamic> json) {
    return Employee(
      id: json['id'] ?? '',
      fullName: json['full_name'] ?? '',
      dob: json['dob'] ?? '',
      fatherName: json['father_name'] ?? '',
      fatherOccupation: json['father_occupation'] ?? '',
      aadhar: json['aadhar'] ?? '',
      address: json['address'] ?? '',
    );
  }
}

File: backend/models/employee.js
const mongoose = require('mongoose');

const employeeSchema = new mongoose.Schema({
  id: { type: String, required: true, unique: true },
  full_name: { type: String, required: true },
  dob: { type: String },
  father_name: { type: String },
  father_occupation: { type: String },
  aadhar: { type: String },
  address: { type: String }
});

module.exports = mongoose.model('profile', employeeSchema);